### PR TITLE
Remove unused webpack files.

### PR DESCRIPTION
**Pull request by rikkert**

### CODE REVIEW

File: docs/general/files.md [hunk 1]
```diff
@@ -30,12 +30,6 @@ Some explanation for a few of the configuration files lying around this boilerpl
 
 * `webpack.base.babel.js`: The shared webpack configuration. In development mode, we add hot reloading and don't minify our code. In production mode, we minify everything.
 
-  * `webpack.dev.babel.js`: Generates the webpack configuration for development.
-
-  * `webpack.prod.babel.js`: Generates the webpack configuration for production.
-
-  * `webpack.test.babel.js`: Generates the webpack configuration for testing.
-
```
Comment (mxstbr) [inline docs/general/files.md]: These files still exist?

**Comment by mxstbr** [comment]
Thanks!

**Comment by rikkert** [comment]
Right, good catch.
Should be correct now.

**Comment by mxstbr** [comment]
This looks good, thanks so much!

**Pull request merged**